hydra-web/src/js/modules/alias.js

```diff
@@ -131,7 +131,7 @@ function(
                     "bVisible":true,
                     "bSearchable":true,
                     "mRender":function(val,type,data){
-                        return val.join(",");
+                        return "<a href='#jobs/"+encodeURIComponent(val)+"/conf'>"+val+"</a>";
```

Comment:
The key point is, when dealing with an array, at the very least you should unwrap it by getting just the first element instead of just using the array directly.